docs/examples/e05_commands_framework/main.py

```diff
@@ -5,12 +5,13 @@
 
 TOKEN = ''
 
-# To setup an extension on the client, just pass it's name to the `Client` constructor.
+# To setup an extension for the client, just pass its name to the `Client` constructor.
 #
 # Each extension has required and optional parameters. Do not forget to pass sufficient parameters for your needs.
-Sakuya = Client(TOKEN,
-    extensions = 'commands_v2',
-    prefix = '!',
+Sakuya = Client(
+    TOKEN,
+    extensions='commands_v2',
+    prefix='!',
 )
```

Comment:
Inconsistent spacing with assignments.

Please use same spacing for parameter assignment as for other variable ones. Even tho stupid code style recommendations describe them differently, they are the same thing.